docs/en/ingest-management/elastic-agent/elastic-agent-encryption.asciidoc

```diff
@@ -0,0 +1,29 @@
+[elastic-agent-encryption]
+= Elastic Agent Configuration Encryption
+
+It is important for users to understand the security model of Elastic Agent and how it handles sensitive values in integration configurations.
+At a high level, Elastic Agent receives configuration data from Fleet Server over an encrypted connection, persists those secrets in an encrypted form on disk.
```

Comment:
```suggestion
At a high level, {agent} receives configuration data from {fleet-server} over an encrypted connection, and persists the configuration in an encrypted form on disk.
```
Do you mean persists the configuration? "persists those secrets" makes it sound like only sensitive info is encrypted.